packages/management/tkg/bundle/config/packageinstalls/tanzu-framework-manager-service-account.yaml

```diff
@@ -0,0 +1,164 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: tanzu-framework-manager-package-sa
+  namespace: tkg-system
+  annotations:
+    kapp.k14s.io/change-group: "tanzu-framework-manager-packageinstall/serviceaccount"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: tanzu-framework-manager-package-role
+  annotations:
+    kapp.k14s.io/change-group: "tanzu-framework-manager-packageinstall/serviceaccount"
+rules:
+  #  RBAC rules to create PackageInstall CR
+  - apiGroups:
+      - ""
+    resources:
+      - secrets
+      - configmaps
+      - serviceaccounts
+      - services
+    verbs:
+      - create
+      - update
+      - get
+      - list
+      - delete
+  - apiGroups:
+      - packaging.carvel.dev
+    resources:
+      - packageinstalls
+    verbs:
+      - create
+      - update
+      - get
+      - list
+      - delete
+  - apiGroups:
+      - rbac.authorization.k8s.io
+    resources:
+      - clusterroles
+      - clusterrolebindings
+    verbs:
+      - create
+      - update
+      - get
+      - list
+      - delete
+  # RBAC rules to install tanzu-framework-manager package
```

Comment:
there might be duplicates in this ClusterRole, merging the rules that are needed by different packages leads to maintenance problems when a developer needs to make changes to RBAC rules of a particular package